app/views/admin/tickets/_form.html.haml

```diff
@@ -12,7 +12,7 @@
       = f.input :title, input_html: { autofocus: true }
       = f.input :description, input_html: { rows: 5, data: { provide: 'markdown-editable' } }
       = f.input :price
-      = f.input :price_currency, as: :select, class: 'form-control', collection: ['USD', 'EUR', 'GBP', 'INR', 'CNY', 'CHF'], include_blank: false
+      = f.input :price_currency, as: :select, class: 'form-control', collection: currencies, include_blank: false
```

Comment:
Then this can be

```ruby
%W(#{ENV['OSEM_DEFAULT_CURRENCIES']})
```

maybe sprinkle in handling of an empty env variable.

Reply:
This seems to produce

irb(main):003:0> %W(#{ENV['OSEM_DEFAULT_CURRENCIES']})
=> ["USD EURO GBP INR CNY"]

irb(main):004:0> ENV['OSEM_DEFAULT_CURRENCIES']
=> "USD EURO GBP INR CNY"

What about this?
`collection: ENV['OSEM_DEFAULT_CURRENCIES'].split`